changelog_unreleased/typescript/12930.md

```diff
@@ -0,0 +1,49 @@
+#### fix formatting for typescript Enum with computed members (#12930 by @HosokawaR)
+
+<!-- prettier-ignore -->
+```tsx
+// Input
+enum A {
+  [i++],
+}
+
+const bar = "bar"
+enum B {
+  [bar] = 2,
+}
+
+const foo = () => "foo";
+enum C {
+  [foo()] = 2,
+}
```

Comment:
I think just the first one example should suffice.

Reply:
Surely. I removed extra examples. 
Thank you !